g2p/cli.py

```diff
@@ -114,23 +136,55 @@ def convert(in_lang, out_lang, input_text, path, debugger):
         output = tg.output_string
         click.echo(output)
 
-@click.option('--mapping', '-m', multiple=True, help="Check a specific mapping, given its output. Please visit http://g2p-studio.herokuapp.com/api/v1/langs for a list of possibilities.")
+
+# Note: with -m eng-ipa, we actually check all the mappings from lang-ipa to eng-ipa.
+@click.option("--list-all", is_flag=True, help="List all mappings that can be specified")
+@click.option("--list-ipa", is_flag=True, help="List IPA mappings that can be specified")
+@click.option(
+    "--mapping", "-m", multiple=True,
+    help="Check specified IPA mapping(s) (default: check all IPA mappings)."
+)
 @cli.command(context_settings=CONTEXT_SETTINGS)
-def doctor(mapping):
-    ''' Check for common errors in mappings.
-        Currently checks for:
+def doctor(mapping, list_all, list_ipa):
+    """ Check for common errors in mappings.
+        There should eventually be more checks here, but doctor currently checks for:
 
         1. Characters that are in IPA mappings but are not recognized by panphon library.
-    '''
+
+        You can list available mappings with --list-all or --list-ipa, or by visiting
+        http://g2p-studio.herokuapp.com/api/v1/langs .
+    """
+    if list_all or list_ipa:
+        out_langs = sorted(set([x["out_lang"] for x in MAPPINGS_AVAILABLE]))
+        if list_ipa:
+            out_langs = [x for x in out_langs if is_ipa(x)]
+        print("Specifying an output language will check all mappings into that language:")
+        for m in out_langs:
+            print(f"{m}:", end="")
+            for n in sorted([x["in_lang"] for x in MAPPINGS_AVAILABLE if x["out_lang"] == m]):
+                print(f" {n}->{m}", end="")
```

Comment:
```suggestion
                print(f" {n}->{m}\n\t", end="")
```

Reply:
I think this makes it a bit more readable. 
![image](https://user-images.githubusercontent.com/14815080/92542188-facccd80-f1fc-11ea-86fc-1bf16e0807dd.png)

Reply:
Agreed, this is much easier to read.

Reply:
@roedoejet I touched up the output a bit. As it was, it looked nice for `--list-ipa`, but not so much for `--list-all`. I'm now using spaces instead of tabs, and counting how many I need to line up each list under its out_lang heading.
Finicky detail I unreasonably care about: I also suppressed the extra whitespace on the blank line between lists.